recipe/meta.yaml

```diff
@@ -29,14 +29,12 @@ requirements:
     - pip
     - cython
     - setuptools
-    - m2w64-toolchain  # [win]
     - libpython  # [win]
     - bzip2  # [not win]
```

Comment:
This is failing because `#  [not win]` should be removed from here.

Reply:
hi @danielnachun thanks for the suggestion! It fully makes sense. However the build still fail with the same error for windows ... any other idea?